flow/include/flow/Platform.h

```diff
@@ -356,7 +356,7 @@ void writeFileBytes(std::filesystem::path const& filename, const uint8_t* data,
 // Write text into file
 void writeFile(std::filesystem::path const& filename, std::string const& content);
 
-std::filesystem::path joinPath(std::filesystem::path const& directory, std::filesystem::path const& filename); // May not even exist tbh, because there's a function that prints out the filename canonical path 
+std::filesystem::path joinPath(std::filesystem::path const& directory, std::filesystem::path const& filename); 
```

Comment:
do we really need a join? we have std::filesystem:concat,  can be just a wrapper..or remove.

Reply:
I don't think it's needed. I'm removing this but as a result quite a few things are failing to build.